lib/libexfat.c

```diff
@@ -668,6 +662,21 @@ int exfat_get_next_clus(struct exfat *exfat, struct exfat_inode *node,
 	return 0;
 }
 
+int exfat_get_inode_next_clus(struct exfat *exfat, struct exfat_inode *node,
```

Comment:
You create this function in this patch. Why ? where is used ? there is no reason in patch description. and if you add new function in patch, next patch should use this.